app/models/group.rb

```diff
@@ -7,8 +7,7 @@ class Group < ApplicationRecord
     presence: true
 
   validates :responsable_contact,
-    :numericality => true,
-    :length => { :minimum => 9, :maximum => 10 }
+    :length => { :minimum => 9 }
```

Comment:
Use the new hash syntax? and keep on the same line?

```
validates :responsable_contact, length: { minimum: 9 }, presence: true
```

I prefer having all the validations for one attribute in one place. So, if you go with this, line 4 can be removed

Reply:
Done in `6be19e2`